functional-tests/fn_bridge_reassignment.py

```diff
@@ -0,0 +1,73 @@
+import time
+
+import flexitest
+
+import testenv
+from constants import UNSPENDABLE_ADDRESS
+from fn_bridge_withdraw_happy import (
+    DEPOSIT_AMOUNT,
+    check_initial_eth_balance,
+    deposit_twice,
+    do_withdrawal,
+    setup_services,
+)
+
+
+@flexitest.register
+class BridgeWithdrawReassignmentTest(testenv.StrataTester):
+    """
+    Makes two DRT deposits, then triggers the withdrawal.
+    The bridge client associated with assigned operator id is stopped.
+    After the dispatch assignment duration is over,
+    Check if new operator is being assigned or not
+    """
+
+    def __init__(self, ctx: flexitest.InitContext):
+        # Example: we spin up 5 operators
+        ctx.set_env(testenv.BasicEnvConfig(101, n_operators=5))
+
+    def main(self, ctx: flexitest.RunContext):
+        address = ctx.env.gen_ext_btc_address()
+        withdraw_address = ctx.env.gen_ext_btc_address()
+        self.debug(f"Address: {address}")
+        self.debug(f"Change Address: {withdraw_address}")
+
+        btc, seq, reth, seqrpc, btcrpc, rethrpc, web3, el_address = setup_services(ctx, self.debug)
+
+        # Check initial balance is 0
+        check_initial_eth_balance(rethrpc, el_address, self.debug)
+
+        # Perform two deposits
+        deposit_twice(ctx, web3, seqrpc, rethrpc, el_address, self.debug)
```

Comment:
I think that there's a way for you to setup an environment with already balances in BTC and Strata, AFAIK @evgenyzdanovich implemented that feature.

Reply:
This aligns with the "minimal resources" principle above.

Reply:
I think we don't have environment with already balances in strata.